Microsoft.Toolkit.Uwp.UI/Brushes/BackdropBlurBrush.cs

```diff
@@ -0,0 +1,108 @@
+﻿// ******************************************************************
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE CODE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+// THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
+// ******************************************************************
+
+// Example brush from https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.media.xamlcompositionbrushbase
+ 
+using Microsoft.Graphics.Canvas.Effects;
+using Windows.UI;
+using Windows.UI.Composition;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Media;
+
+namespace Microsoft.Toolkit.Uwp.UI.Brushes
+{
+    /// <summary>
+    /// The <see cref="BackdropBlurBrush"/> is a <see cref="Brush"/> that blurs whatever is behind it in the application.
+    /// </summary>
+    public class BackdropBlurBrush : XamlCompositionBrushBase
+    {
+        /// <summary>
+        /// Identifies the <see cref="Amount"/> dependency property.
+        /// </summary>
+        public static readonly DependencyProperty AmountProperty = DependencyProperty.Register(
+            nameof(Amount),
+            typeof(double),
+            typeof(BackdropBlurBrush),
+            new PropertyMetadata(0.0, new PropertyChangedCallback(OnAmountChanged)));
+
+        /// <summary>
+        /// Gets or sets the amount of gaussian blur to apply to the background.
+        /// </summary>
+        public double Amount
+        {
+            get { return (double)GetValue(AmountProperty); }
+            set { SetValue(AmountProperty, value); }
+        }
+
+        private static void OnAmountChanged(DependencyObject d, DependencyPropertyChangedEventArgs e)
+        {
+            var brush = (BackdropBlurBrush)d;
+
+            // Unbox and set a new blur amount if the CompositionBrush exists.
+            brush.CompositionBrush?.Properties.InsertScalar("Blur.BlurAmount", (float)(double)e.NewValue);
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="BackdropBlurBrush"/> class.
+        /// </summary>
+        public BackdropBlurBrush()
+        {
+            this.FallbackColor = Colors.Transparent;
```

Comment:
**This** should be removed.

Reply:
Transparent should already be the default, so you should be able to remove the entire line

Reply:
Yeah, I realized that over the weekend, fixing now.